website/docs/plugin/framework/resources/update.mdx

```diff
@@ -6,25 +6,25 @@ description: >-
 
 # Update Resources
 
-In-place update is part of the basic Terraform lifecycle for managing resources. During the [`terraform apply` command](/cli/commands/apply), Terraform calls the provider [`ApplyResourceChange`]((/plugin/framework/internals/rpcs#applyresourcechange-rpc)) RPC, in which the framework calls the [`resource.Resource` interface `Update` method](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource#Resource.Update). The request contains Terraform prior state, configuration, and plan data. The response contains updated state data. The data is defined by the [schema](/plugin/framework/schemas) of the resource.
+In-place update is part of the basic Terraform lifecycle for managing resources. During the [`terraform apply` command](/terraform/cli/commands/apply), Terraform calls the provider [`ApplyResourceChange`](https://terraform.io/(/plugin/framework/internals/rpcs#applyresourcechange-rpc)) RPC, in which the framework calls the [`resource.Resource` interface `Update` method](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource#Resource.Update). The request contains Terraform prior state, configuration, and plan data. The response contains updated state data. The data is defined by the [schema](/terraform/plugin/framework/schemas) of the resource.
```

Comment:
🔴 Same here: https://github.com/hashicorp/terraform-plugin-framework/pull/651/files#r1097477686

Reply:
Will be fixed here: https://github.com/hashicorp/terraform-plugin-framework/pull/655